app/routes/events_.$eventId.tsx

```diff
@@ -0,0 +1,86 @@
+import type { LoaderFunction } from '@remix-run/node';
+import { useParams, useLoaderData, Link } from '@remix-run/react';
+import { db } from '~/modules/database/db.server';
+import { json } from '@remix-run/node';
+import { eventDataPatcher } from '~/modules/events/event';
+
+export const loader: LoaderFunction = async ({ params }) => {
```

Comment:
I noticed that we mix using arrow functions and function declarations to define our loaders:

```ts 
export const loader: LoaderFunction = async (args) => {}
```

vs

```ts 
export async function loader(args: LoaderFunctionArgs) {}
```

I think it may be a good idea to choose one and stick with it for consistency in our codebase. 

Also, I'm not sure why this is the case, but using the arrow function method seems to cause `useLoaderData` to return `any` whereas using the a function declaration returns the expected type.

Reply:
I like the function one is more inline with code base, will switch to it.